kong/plugins/session/migrations/003_330_to_3100.lua

```diff
@@ -0,0 +1,22 @@
+return {
+  postgres = {
+    up = [[
+      CREATE TABLE IF NOT EXISTS session_metadatas(
+        id            uuid,
+        session_id    uuid                    REFERENCES "sessions" ("id") ON DELETE CASCADE,
+        sid           text UNIQUE,
+        subject       text,
```

Comment:
How about creating a composite index with `subject` and `audience` as they are supposed to work as search keyword in select query. DB will have a poor performance for such select SQL when there are too much entries in metadata table.

Reply:
Good idea.